core/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -462,6 +465,33 @@ public void simpleStats() throws Exception {
         assertThat(stats.getTotal().getIndexing(), nullValue());
         assertThat(stats.getTotal().getGet(), nullValue());
         assertThat(stats.getTotal().getSearch(), nullValue());
+
+        // index failed
+        try {
+            client().prepareIndex("test1", "type1", Integer.toString(1)).setSource("field", "value").setVersion(1)
+                    .setVersionType(VersionType.EXTERNAL).execute().actionGet();
```

Comment:
I'd stick a `fail("Expected a version conflict");` right here just to make sure the failure occurs.